### PR TITLE
Update device.py

### PR DESCRIPTION
**Pull request by svh1985**
Added support for the magnetic door sensor which comes with Nuki 2.0:
https://developer.nuki.io/page/nuki-bridge-http-api-1-11/4/#heading--doorsensor-states

Ran a test against my bridge:
```
from pynuki import NukiBridge

bridges = NukiBridge.discover()
br = bridges[0]
br.token = "MY_TOKEN"

print(br.devices[0].door_sensor_state_name)
print(br.devices[0].door_sensor_state)
```
Output door closed:
```
door closed
2
```
Output door opened:
```
door opened
3
```

### CODE REVIEW

File: pynuki/device.py [hunk 1]
```diff
@@ -33,6 +33,14 @@ def state(self):
     @property
     def state_name(self):
         return self._json.get("stateName")
+    
+    @property
```
Comment (pschmitt) [inline pynuki/device.py:37]: ATM the `device` class is not the right place for this. It was meant as a base class that hosts all the common attributes. You mentioned this is for Niki 2.0 devices only. It  would make more sense to create a new `NukiV2` that inherits from `NukiLock` or even better: an intermediate class `NukiBaseLock` and a `NukiV1` and `NukiV2` that both inherit from it.

Comment (chpego) [reply pynuki/device.py:37]: Yes, seems to be good idea, how i can help/improve you to build code ?

Comment (pschmitt) [reply pynuki/device.py:37]: I'm not sure what you mean. What helps most are PRs :)

**Comment by pschmitt** [comment]
Closing as #61 has been merged.

**Pull request closed without merging**